src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task{
+
```

Comment:
This class has a empty line before its attributes. However I noticed other classes do not have this empty line. Try and make it more consistent